### ISSUE TITLE
Error running in flutter ios

### ISSUE THREAD

**Issue by shraddhashakya** [Instabug/Instabug-Flutter#324]
```
Xcode's output:
↳
    Writing result bundle at path:
        /var/folders/tn/w60jd_wn3tg9020v2spcdyl00000gn/T/flutter_tools.1IpQ3g/flutter_ios_build_temp_dirQnmPE8/temporary_xcresult_bundle

    /Users/Documents/development/flutter/.pub-cache/hosted/pub.dartlang.org/instabug_flutter-11.6.0/ios/Classes/Modules/FeatureRequestsApi.m:13:6: error: definition of 'IBGFeatureRequests' must
    be imported from module 'Instabug.IBGFeatureRequests' before it is required
        [IBGFeatureRequests show];
         ^
    In module 'Instabug' imported from /Users/Documents/development/flutter/.pub-cache/hosted/pub.dartlang.org/instabug_flutter-11.6.0/ios/Classes/Util/ArgsRegistry.h:2:
    /Users/StudioProjects/next21/build/ios/Debug-development-iphonesimulator/XCFrameworkIntermediates/Instabug/Instabug.framework/Headers/IBGFeatureRequests.h:16:12: note: definition here is not
    reachable
    @interface IBGFeatureRequests : NSObject
               ^
    1 error generated.
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in dependency order
    /Users/StudioProjects/next21/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 15.2.99. (in target 'AppAuth' from project 'Pods')
    /Users/StudioProjects/next21/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target
    versions is 9.0 to 15.2.99. (in target 'FMDB' from project 'Pods')

    Result bundle written to path:
        /var/folders/tn/w60jd_wn3tg9020v2spcdyl00000gn/T/flutter_tools.1IpQ3g/flutter_ios_build_temp_dirQnmPE8/temporary_xcresult_bundle


Modules Issue (Xcode): Definition of 'IBGFeatureRequests' must be imported from module 'Instabug.IBGFeatureRequests' before it is required
/Users//Documents/development/flutter/.pub-cache/hosted/pub.dartlang.org/instabug_flutter-11.6.0/ios/Classes/Modules/FeatureRequestsApi.m:12:5
```

**Comment by DavidMina96** [comment]
Hi @shraddhashakya, 

Thank you for reporting this; it's most likely a caching issue at your end. You can try running the below commands and then run the app again:
1. `flutter clean`
2. `pod cache clean --all` inside your `ios` directory followed by `pod install`

Please feel free to re-open the issue if this doesn't resolve it for you.

**Comment by romk1n** [comment]
we have the same issue above steps including `git clean -ffdx` didn't help is there another workaround ?

**Comment by matfisch** [comment]
We had the same issue and resolved it by upgrading Xcode from 13.2.x -> 13.3.x on our machines and CI environments.
